src/exe/Main.hs

```diff
@@ -68,6 +69,11 @@ import           HST.Environment.FreshVars      ( PMState(PMState)
 import qualified HST.Frontend.FromHSE          as FromHSE
 import qualified HST.Frontend.ToHSE            as ToHSE
 
+-- | A data type for all frontends that can be used for parsing the given input
+--   program in @haskell.src.transformations@.
```

Comment:
I suggest using "front end" or "front-end" for comments and strings, because "frontend" is not included in many dictionaries.
```suggestion
-- | A data type for all front ends that can be used for parsing the given input
--   program in @haskell-src-transformations@.
```